src/Build/Instance/ProjectInstance.cs

```diff
@@ -10,6 +10,7 @@
 using System.IO;
 using System.Linq;
 using System.Runtime.CompilerServices;
+using System.Threading;
```

Comment:
nit: This namespace import looks unused.

Reply:
Thanks. Removed.